### DISCUSSION TITLE
Feature: Export to Wezterm

### DISCUSSION THREAD

**Discussion by mrjones2014** [olimorris/onedarkpro.nvim#179] [Ideas]
I see terminal themes are now done via an export of your current theme, this is great and an improvement over how most themes handle it!

Could we get a Wezterm export though? 🥺

Example themes can be found here: https://github.com/wez/wezterm/tree/main/assets/colors

**Comment by olimorris** [comment]
Great shout! I've added a wezterm theme in the latest commit. Let me know what you think.

**Reply by mrjones2014** [reply]
Works great, thank you!
